src/Integration.Vsix/Analysis/AnalysisStrings.resx

```diff
@@ -163,4 +163,13 @@
   <data name="Settings_UsingDefaultSettings" xml:space="preserve">
     <value>Default settings will be used</value>
   </data>
+  <data name="Notifier_AnalysisFinished" xml:space="preserve">
+    <value>SonarLint: Finished analyzing {0}</value>
+  </data>
+  <data name="Notifier_AnalysisFailed" xml:space="preserve">
+    <value>SonarLint: Failed to analyze {0}. See the Output Window for more information.</value>
+  </data>
+  <data name="Notifier_AnalysisStarted" xml:space="preserve">
+    <value>SonarLint: Analyzing {0}</value>
```

Comment:
Minor: should this message end with an ellipsis ("...") to indicate it is an ongoing action?
I'm trying to remember if this is a general convention.

Reply:
I don't think so, the MS's one doesn't have it. 
(But then again, they have a spinner right next to it 🤷 we also have the progress icon but it's not as noticeable)